@bazel/webpack/webpack/worker/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+alias(
```

Comment:
Can you leave a comment in this file stating why this aliasing is necessary?

Reply:
Thank you!